src/core/endpoints/PlayerEndpoint.ts

```diff
@@ -43,7 +44,7 @@ export function build(opts: PlayerEndpointOptions): IPlayerRequest {
       client: opts.client,
       playlistId: opts.playlist_id,
       // Workaround streaming URLs returning 403 or getting throttled when using Android based clients.
-      params: is_android ? '2AMBCgIQBg' : opts.params
+      params: is_android ? encodeShortsParam() : opts.params
```

Comment:
In yt-dlp, we were using using `{1: {2: 6}} -> CgIQBg==` without `59: 1`. As you discovered, changing it to `{1: {1: 1}} -> CgIQBg==` is working. Thanks for the finding!

But could you pls explain what the `59: 1` is for?

Reply:
Hi! `59: 1` is so we can still get `storyboards` and a few other fields I can't remember right now.

Reply:
hm.. we are getting storyboards without it